developers/contributor-guide/current/weaviate-contributors-guide/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: layout-documentation
+solution: contributor-guide
+sub-menu: Weaviate Contributor's Guide
+title: Weaviate Contributor's Guide
+intro: This guide contains all of the resources you will need to contribute to Weaviate..
+description: This guide contains all of the resources you will need to contribute to Weaviate.
+tags: ['index', 'contributor-guide']
+menu-order: 0
+open-graph-type: article
+og-img: documentation.jpg
+toc: true
+redirect_from:
+---
+
+## Why to contribute?
+
+A project like Weaviate is excellent opportunity to gain experience with Vector Search Engines, Docker, Kubernetes, GraphQL, REST and various other language clients. You will work on the forefront of search, ML/AI and cloud-native technologies, but most of all, you be part of an amazing open source project. You will also get amazing networking opportunities with experienced and like-minded software engineers.
```

Comment:
- `A project like Weaviate` should be condensed to `Weaviate`
- `Vector Search Engines` should be lowercase